jfrog_platform/README.md

```diff
@@ -0,0 +1,64 @@
+# Artifactory and Xray Logging Analytics with Datadog
+
+## Overview
+
+The following describes how to configure Datadog to gather metrics from Artifactory and Xray.
+
+## Requirements
```

Comment:
```suggestion
## Setup

### Requirements
```

I believe the Setup header is required as it marks the beginning of `Configuration` tab in the tiles.